src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
+$font-family: Roboto, sans-serif;
+$text-color: #060b35;
+$secondary-color: #616070;
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  justify-content: center;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.product {
```

Comment:
it's a separate BEM block - according to the BEM rules it should be in a separate file